config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  namespace 'api' do
+    namespace 'v0' do
```

Comment:
looks weird. but maybe it makes sense for the first demo.
@mrgrecha what do you think?

Reply:
"The compact style is only forced for classes/modules with one child."
Rubocop offense too

Reply:
@alexandr-senyuk I agree with you. @Hawatz I recommend you create `v1` version. As you don't have other clients(mobile, other apps and etc) we can use just v1 and update it when necessary.